mdbenchmark/analyze.py

```diff
@@ -182,19 +103,27 @@ def analyze(directory, plot, ncores, output_name):
     df.to_csv(output_name)
 
     if plot:
+        console.warn(
+            'This feature is now outdated.'
+            'Please use mdbenchmark plot to access all plotting features'
```

Comment:
```
'Please use `mdbenchmark plot` to ...
```

Reply:
We should make up our mind here. We do not use a common style for these messages. In general all commands should be defined via `kwargs`, such that they are shown in bold letters. We could think about also hyphenating them, to aid users with terminals that do not support colors.